src/components/RadioInputGroup/RadioInputGroup.test.tsx

```diff
@@ -0,0 +1,361 @@
+    label: 'Test label',
+    options: testOptions,
+    onChange: () => {},
+  };
+
+  it('should render without crashing', () => {
+    render(<RadioInputGroup {...defaultProps} />);
+    expect(screen.getByRole('radiogroup')).toBeInTheDocument();
+  });
+
+  it('should render the provided options', () => {
+    render(<RadioInputGroup {...defaultProps} />);
+    defaultProps.options.forEach((option) => {
+      expect(screen.getByText(option.label)).toBeInTheDocument();
+    });
+  });
+
+  it('should make the option with the provided checkedOptionId checked', () => {
+    const checkedOption = defaultProps.options[1]!;
+    render(
+      <RadioInputGroup {...defaultProps} checkedOptionId={checkedOption?.id} />,
+    );
+    expect(screen.getByText(checkedOption.label)).toBeChecked();
+  });
+
+  it('should not make any option checked if no checkedOptionId is provided', () => {
+    render(<RadioInputGroup {...defaultProps} />);
+    defaultProps.options.forEach((option) => {
+      expect(screen.getByText(option.label)).not.toBeChecked();
+    });
+  });
+
+  it('should set the aria-activedescendant attribute to the provided checkedOptionId', () => {
+    const checkedOption = defaultProps.options[1]!;
+    render(
+      <RadioInputGroup {...defaultProps} checkedOptionId={checkedOption?.id} />,
+    );
+    expect(screen.getByRole('radiogroup')).toHaveAttribute(
+      'aria-activedescendant',
+      checkedOption.id,
+    );
+  });
+
+  it('should not set the aria-activedescendant attribute if the provided checkedOptionId is not an id of the provided options', () => {
+    render(
+      <RadioInputGroup {...defaultProps} checkedOptionId="incorrect-id" />,
+    );
+    expect(screen.getByRole('radiogroup')).not.toHaveAttribute(
+      'aria-activedescendant',
+    );
+  });
+
+  it('should pass down the provided onChange handler to the radio input options', async () => {
+    const user = userEvent.setup();
+    const clickedOption = defaultProps.options[2]!;
+    const onChangeMock = jest.fn();
+    render(<RadioInputGroup {...defaultProps} onChange={onChangeMock} />);
+    expect(onChangeMock).not.toHaveBeenCalled();
+
+    await user.click(screen.getByText(clickedOption.label));
+    expect(onChangeMock).toHaveBeenCalled();
+  });
+
+  describe('User events', () => {
+    it('should become focussed when user tabs on it', async () => {
+      const user = userEvent.setup();
+      render(<RadioInputGroup {...defaultProps} />);
+
+      await user.tab();
+      expect(screen.getByRole('radiogroup')).toHaveFocus();
+    });
+
+    it('should loose focus when user tabs away from it', async () => {
+      const user = userEvent.setup();
+      render(<RadioInputGroup {...defaultProps} />);
+
+      await user.tab();
+      await user.tab();
+      expect(screen.getByRole('radiogroup')).not.toHaveFocus();
+    });
+
+    it(`should call the onChange callback with the next option on pressing ${KeyCodes.ARROW_RIGHT}`, async () => {
```

Comment:
They are already under `describe('User events'`. Then give a suggestion

Reply:
Never mind the test descriptions. That's WIP
![user events](https://user-images.githubusercontent.com/66679828/209645398-59a3ae59-839e-4554-a67e-08d9d5ef4db1.png)

Reply:
Also - please, don't mark conversations as `resolved` before I've looked at your responses. Otherwise, I can miss your responses, because those conversations are no longer visible to me.